lib/internal/util/comparisons.js

```diff
@@ -1,14 +1,32 @@
 'use strict';
 
 const {
+  Array,
+  ArrayBuffer,
   ArrayIsArray,
   ArrayPrototypeFilter,
   ArrayPrototypePush,
+  BigInt,
+  BigInt64Array,
   BigIntPrototypeValueOf,
+  BigUint64Array,
+  Boolean,
   BooleanPrototypeValueOf,
+  DataView,
+  Date,
   DatePrototypeGetTime,
   Error,
+  Float16Array,
```

Comment:
https://github.com/nodejs/node/blob/4fd5db479a8b37a48c565a79eeb719a0fb459469/lib/eslint.config_partial.mjs#L359-L363
```suggestion
  globalThis: { Float16Array },
```